src/_common/__tests__/Clock-test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import expect from 'expect';
+import { shallow,render } from 'enzyme';
+import { FormattedTime } from 'react-intl';
+import Clock from '../Clock';
+
+describe('Clock',()=>{
+	const wrapper = shallow(<Clock time={ new Date()}> </Clock>);
+    const time = new Date(wrapper.node.props.value);
+	it("The day is Today's day ",()=>{
```

Comment:
Agree, think a lot on the description, it is very imortant.
Also, in general the now time is not a good idea to depend on something that 'ticks'. In rare cases possible for some time to have passed and the times to differ.
Also consider running the const wrapper and const time inside the it() block. Tests should be very isolated from each other.